### PR TITLE
Fix build

### PR DESCRIPTION
**Pull request by abdonrd**
The `buildRoot` was reversed.

Also, added missed require.

### CODE REVIEW

**Comment by chuckh** [comment]
LGTM. Thanks @abdonrd 👍

**Comment by abdonrd** [comment]
Your welcome! :)

File: generators/app/gulpfile.js [hunk 1]
```diff
@@ -11,6 +11,7 @@
 'use strict';
 
 const path = require('path');
+const process = require('process');
```
Comment (robdodson) [inline generators/app/gulpfile.js]: I could be mistaken but I don't think you have to require process. It's global https://nodejs.org/api/globals.html#globals_process

Not sure what the recommended best practice is

Comment (abdonrd) [inline generators/app/gulpfile.js]: Hum! Probably you are right...

Comment (abdonrd) [inline generators/app/gulpfile.js]: https://github.com/PolymerElements/generator-polymer-init-custom-build/pull/20

**Pull request merged**